Docs/Configuration.tex

```diff
@@ -2654,6 +2654,19 @@ \subsection{Quirks Properties}\label{kernelpropsquirks}
   link-down state infinitely and a Wi-Fi adapter may fail to discover networks.
   Gigabyte is one of the most common OEMs with these issues.
 
+\item
+  \texttt{DisableIoMapperMapping}\\
+  \textbf{Type}: \texttt{plist\ boolean}\\
+  \textbf{Failsafe}: \texttt{false}\\
+  \textbf{Requirement}: 13.3 (not required for older)\\
+  \textbf{Description}: Disables mapping PCI bridge device memory in IOMMU (VT-d),
```

Comment:
In my opinion, just state `Disables mapping PCI bridge device memory in IOMMU (VT-d)` in description, and leave the useful scenario as a side note below.

Reply:
Will move the second part of that description to `Note 1` and change existing `Note 1` to `Note 2`